build.gradle.kts

```diff
@@ -35,16 +38,37 @@ java {
     withSourcesJar()
 }
 
-tasks.withType<Javadoc> { exclude("/com/mongodb/hibernate/internal/**") }
+tasks.withType<Javadoc> {
+    exclude("/com/mongodb/hibernate/internal/**")
+
+    val standardDocletOptions = options as StandardJavadocDocletOptions
+    standardDocletOptions.apply {
+        author(true)
+        version(true)
+        encoding = "UTF-8"
+        charSet("UTF-8")
+        docEncoding("UTF-8")
+        addBooleanOption("html5", true)
+        addBooleanOption("-allow-script-in-comments", true)
+        links =
+            listOf(
+                "https://docs.oracle.com/en/java/javase/17/docs/api/",
+                "https://docs.jboss.org/hibernate/orm/6.6/javadocs/",
+                "https://mongodb.github.io/mongo-java-driver/5.3/apidocs/mongodb-driver-sync/")
+    }
+}
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 // Integration Test
 
+// Added `Action` explicitly due to an intellij 2025.2 false positive: https://youtrack.jetbrains.com/issue/KTIJ-34210
```

Comment:
Not related to publishing but clears the intellij warning in the latest versions of intellij.